### ISSUE TITLE
An app's production build breaks if source maps are enabled.

### ISSUE THREAD

**Issue by ssylvia** [Esri/ember-esri-loader#35]
When I run `ember build -e=production` on an app that has `ember-esri-loader` installed, I get the following error:

```sh
Error: ENOENT: no such file or directory, open 'project-path/esri-loader-test/tmp/uglify_writer-input_base_path-GKD3EADg.tmp/assets/dist/esri-loader.min.js.map'
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:491:33)
    at UglifyWriter.processFile (project-path/esri-loader-test/node_modules/broccoli-uglify-sourcemap/index.js:101:41)
    at project-path/esri-loader-test/node_modules/broccoli-uglify-sourcemap/index.js:65:16
    at Array.forEach (native)
    at project-path/esri-loader-test/node_modules/broccoli-uglify-sourcemap/index.js:55:25
    at Array.forEach (native)
    at UglifyWriter.build (project-path/esri-loader-test/node_modules/broccoli-uglify-sourcemap/index.js:54:19)
    at project-path/esri-loader-test/node_modules/broccoli-plugin/read_compat.js:93:34
    at tryCatch (project-path/esri-loader-test/node_modules/rsvp/dist/rsvp.js:525:12)

The broccoli plugin was instantiated at:
    at Fingerprint.Plugin (project-path/esri-loader-test/node_modules/broccoli-plugin/index.js:7:31)
    at Fingerprint.Filter [as constructor] (project-path/esri-loader-test/node_modules/broccoli-filter/index.js:34:10)
    at new Fingerprint (project-path/esri-loader-test/node_modules/broccoli-asset-rev/lib/fingerprint.js:21:10)
    at Fingerprint (project-path/esri-loader-test/node_modules/broccoli-asset-rev/lib/fingerprint.js:16:12)
    at new AssetRev (project-path/esri-loader-test/node_modules/broccoli-asset-rev/lib/asset-rev.js:31:25)
    at AssetRev (project-path/esri-loader-test/node_modules/broccoli-asset-rev/lib/asset-rev.js:7:12)
    at Class.postprocessTree (project-path/esri-loader-test/node_modules/broccoli-asset-rev/index.js:31:40)
    at projectOrAddon.addons.reduce (project-path/esri-loader-test/node_modules/ember-cli/lib/utilities/addon-process-tree.js:6:25)
    at Array.reduce (native)
    at addonProcessTree (project-path/esri-loader-test/node_modules/ember-cli/lib/utilities/addon-process-tree.js:4:32)
```

Thought it might have something to do with this issue: https://github.com/emberjs/ember.js/pull/15720, but I updated to my app to ember v2.16.0 but still get the error.

Here's a sample app: https://github.com/ssylvia/ember-esri-loader-sourcemap-build-fail-demo.

**Comment by tomwayson** [comment]
Thanks @ssylvia 

Did you already try [excluding esri-oader](https://ember-cli.com/asset-compilation#exclude-from-minification) in that app?

**Comment by tomwayson** [comment]
I was able to repro in the dummy app by copying the source map config from your app:

```js
    sourcemaps: {
      enabled: true,
      extensions: ['js'],
    },
```

I was then able to fix the build error by excluding esri-loader from minification with:

```js
    minifyJS: {
      options: {
        exclude: ["**/esri-loader.min.js"]
      }
    }
```

That's at least a workaround for now.

**Comment by tomwayson** [comment]
@ssylvia FYI - I have fixed this issue in esri-loader@1.5.2, but there are still some sourcemap issues in ember-esri-loader, so I won't be cutting a new release of this repo until I've had time to work those out. In the mean time you can bump esri-loader.

**Comment by tomwayson** [comment]
@ssylvia this is resolved in v0.2.3

**Comment by ssylvia** [comment]
Thanks @tomwayson!
